tests/queries/0_stateless/02891_rename_table_without_keyword.sql

```diff
@@ -0,0 +1,14 @@
+DROP DATABASE IF EXISTS rename_db;
```

Comment:
It might worth to add some "negative" test cases:
1. `RENAME rename_db TO rename_db_2`
2. `RENAME <dictionary name> TO rename_dictionary_2`

Reply:
done